hunts/models.py

```diff
@@ -22,6 +26,15 @@ def save(self, *args, **kwargs):
             self.slug = slugify(self.name)
         super().save(*args, **kwargs)
 
+    def clean(self):
```

Comment:
I think it is cleaner to override the `save()` method so you can avoid calling `save()`, `clean()`, and `save()` again in succession.